<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heydemann</class>
 <widget class="QWidget" name="heydemann">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCALabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:MonoName</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Heydemann Correction</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:set_ikHeydemann</string>
   </property>
   <item>
    <property name="text">
     <string>On</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Acquisition</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:HeySelection</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetHeySelection</string>
   </property>
   <item>
    <property name="text">
     <string>Andreas</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Energy(eV)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mirror(arcsec)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grating(arcsec)</string>
    </property>
   </item>
  </widget>
  <widget class="QCAPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:IK_AKTION</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liSTATUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>MEASURE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Flags</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liFLAGS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCALabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 127);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbbiFLAGS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>250</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>acquired</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>270</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liVal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Evaluation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>View Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IK-Channel</string>
   </property>
  </widget>
  <widget class="QCAComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:mbboHeydSignal</string>
   </property>
   <item>
    <property name="text">
     <string>0 (M)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 (M)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 (M)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 (M)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 (G)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 (G)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 (G)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 (G)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>fb-Details</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:liVAL</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="startAxis" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>230</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Start</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:HeyStart</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetHeyStart</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="stopAxis" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>230</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Stop</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:HeyStop</string>
   </property>
   <property name="units" stdset="0">
    <string>eV</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetHeyStop</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="offAxis" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>230</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Offset</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:HeyOffset</string>
   </property>
   <property name="units" stdset="0">
    <string/>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetHeyOffset</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="rateAxis" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>230</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="description" stdset="0">
    <string>Rate</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:HeyRate</string>
   </property>
   <property name="units" stdset="0">
    <string>Hz</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:SetHeyRate</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
  <customwidget>
   <class>QCAComboBox</class>
   <extends>QComboBox</extends>
   <header>qcacombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
